docs/ops/detection/DeformablePSROIPooling_1.md

```diff
@@ -76,18 +83,50 @@ The box coordinates are specified as five element tuples: *[batch_id, x_1, y_1,
 
 **Inputs**:
 
-*   **1**: 4D input tensor with feature maps. Required.
+*   **1**: 4D input tensor of type *T* and shape `[N_in, C_in, H_in, W_in]` with position sensitive score map. Required.
 
-*   **2**: 2D input tensor describing box consisting of five element tuples: `[batch_id, x_1, y_1, x_2, y_2]`. Required.
+*   **2**: 2D input tensor of type *T* and shape `[NUM_ROIS, 5]`. It contains a list of five element tuples describing a single ROI (region of interest): `[batch_id, x_1, y_1, x_2, y_2]`. Required.
+Batch indices must be in the range of `[0, N_in-1]`.
 
-*   **3**: 4D input blob with transformation [values](https://arxiv.org/abs/1703.06211) (offsets). Optional.
+*   **3**: 4D input tensor of type *T* and shape `[NUM_ROIS, 2*NUM_CLASSES, group_size, group_size]` with transformation values. It contains normalized `[0, 1]` offsets for each ROI bin left top corner coordinates. Channel dimension is multiplied by `2` because of encoding two `(x, y)` coords. Optional.
 
 **Outputs**:
+*   **1**: 4D output tensor of type *T* shape `[NUM_ROIS, output_dim, group_size, group_size]` with ROIs score maps.
+
+**Types**:
 
-*   **1**: 4D output tensor with areas copied and interpolated from the 1st input tensor by coordinates of boxes from the 2nd input and transformed according to values from the 3rd input.
+* *T*: Any floating point type.
+ 
 
 **Example**
 
+* Two inputs (without offsets)
+```xml
+<layer id="6" name="rfcn_cls_offset" type="DeformablePSROIPooling" version="opset1">
+			<data spatial_scale="0.0625" output_dim="882" group_size="3" mode="bilinear_deformable" spatial_bins_x="4" spatial_bins_y="4" pooled_width="3" pooled_height="3" trans_std="0.0" part_size="3"/>
```

Comment:
Please, fix indentation. Use 4 spaces instead of "tab" characters.

Reply:
Sure, done.